scripts/manifest

```diff
@@ -7,7 +7,7 @@ fi
 
 set -e -x
 
-export REPO="rancher/k3s-upgrade"
+export REPO="k3s-io/k3s-upgrade"
```

Comment:
We are still only publishing under the rancher org in docker hub. is this the repo for that?

Reply:
Edited in error. Fixed.